src/systemctl/systemctl-show.c

```diff
@@ -64,7 +64,7 @@ typedef struct ExecStatusInfo {
 
         ExecCommandFlags flags;
 
-        LIST_FIELDS(struct ExecStatusInfo, exec);
+        LIST_FIELDS(struct ExecStatusInfo, exec_status_infos);
```

Comment:
"infos" is a weird thing too. not a proper word. When I read it like this I'd assume a German tried to write English here, and didn't really succeed...

(becuase "infos" is a German word, but not an English one)

Hence, ehre too i'd leave the status quo ante in place, but if you insist renaming it, maybe suffix with "_list"